tensorflow/compiler/xla/service/gpu/cholesky_thunk.cc

```diff
@@ -61,13 +61,13 @@ Status CholeskyThunk::ExecuteOnStream(const ExecuteParams& params) {
        << " workspace=" << workspace_buffer_.ToString()
        << " info=" << info_buffer_.ToString();
 
-  CusolverContext* context;
+  GpusolverContext* context;
```

Comment:
Probably `GpuSolver`?

Reply:
thought about making the `s` uppercase, but kept it as is to be consistent with the underlying lowercase `s` in `cusolver` and `rocsolver` ... will push out a commit to make it upper case.